.travis.yml

```diff
@@ -42,6 +42,7 @@ matrix:
         # Sphinx is pinned so that new versions that introduce new warnings won't suddenly cause build failures.
         # (Updating the version is fine as long as no warnings are raised by doing so.)
         - python -m pip install sphinx~=1.6.1
+        - python -m pip install blurb
```

Comment:
Well, blurb is under development but is the 1.0.2 release specifically under development? My assumption is you mean the former and not the latter, which means I can push a 1.0.2 release so that the changes in that bugfix release go public because right now no one is getting access since pip considers a dev release unusable without opting in.